package com.kwad.sdk.core.b.a;

import org.json.JSONObject;

/* JADX INFO: loaded from: classes3.dex */
public final class gb implements com.kwad.sdk.core.d<com.kwad.sdk.core.webview.b.a.a> {
    @Override // com.kwad.sdk.core.d
    public final /* bridge */ /* synthetic */ void a(com.kwad.sdk.core.b bVar, JSONObject jSONObject) {
        a((com.kwad.sdk.core.webview.b.a.a) bVar, jSONObject);
    }

    @Override // com.kwad.sdk.core.d
    public final /* bridge */ /* synthetic */ JSONObject b(com.kwad.sdk.core.b bVar, JSONObject jSONObject) {
        return b((com.kwad.sdk.core.webview.b.a.a) bVar, jSONObject);
    }

    private static void a(com.kwad.sdk.core.webview.b.a.a aVar, JSONObject jSONObject) {
        if (jSONObject == null) {
            return;
        }
        aVar.bcL = jSONObject.optString("Access-Control-Allow-Origin");
        if (JSONObject.NULL.toString().equals(aVar.bcL)) {
            aVar.bcL = "";
        }
        aVar.bcM = jSONObject.optString("Timing-Allow-Origin");
        if (JSONObject.NULL.toString().equals(aVar.bcM)) {
            aVar.bcM = "";
        }
        aVar.bcN = jSONObject.optString(com.alipay.sdk.m.t.e.f);
        if (JSONObject.NULL.toString().equals(aVar.bcN)) {
            aVar.bcN = "";
        }
        aVar.bcO = jSONObject.optString("Date");
        if (JSONObject.NULL.toString().equals(aVar.bcO)) {
            aVar.bcO = "";
        }
    }

    private static JSONObject b(com.kwad.sdk.core.webview.b.a.a aVar, JSONObject jSONObject) {
        if (jSONObject == null) {
            jSONObject = new JSONObject();
        }
        if (aVar.bcL != null && !aVar.bcL.equals("")) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, "Access-Control-Allow-Origin", aVar.bcL);
        }
        if (aVar.bcM != null && !aVar.bcM.equals("")) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, "Timing-Allow-Origin", aVar.bcM);
        }
        if (aVar.bcN != null && !aVar.bcN.equals("")) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, com.alipay.sdk.m.t.e.f, aVar.bcN);
        }
        if (aVar.bcO != null && !aVar.bcO.equals("")) {
            com.kwad.sdk.utils.ac.putValue(jSONObject, "Date", aVar.bcO);
        }
        return jSONObject;
    }
}
